man/man4/plumber.4

```diff
@@ -23,8 +26,8 @@ in the format of
 Its services are posted via
 .IR 9pserve (4)
 as
-.BR plumb .
-and consist of two
+.BR plumb ,
+and consists of two
```

Comment:
The subject is plural, so the predicate should remain to be “consist”, shouldn’t it?

Reply:
You are right.